benchmark.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Benchmark Drupal site.
+
+set -xeuo pipefail
+export PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:$HOME/.composer/vendor/bin"
+
+cookie_file="cookies.txt"
+
+# Get login URL and remove \r fron the end.
+login_url="$(lando drush uli)"
+login_url="${login_url//[$'\r']}"
+
+# Log in and save cookie info to file.
+curl \
+  --insecure \
+  --max-redirs 5 \
+  --cookie-jar $cookie_file \
```

Comment:
Apparently it would be possible:
> If you set the file name to a single dash, "-", the cookies will be written to stdout.
From https://curl.se/docs/manpage.html.
Could you @hkirsman change this to use a variable in the script instead of a file so that running the script is not dependent on a writable filesystem?